src/imu/sensors/magnetometer.cpp

```diff
@@ -0,0 +1,100 @@
+#include "magnetometer.hpp"
+
+namespace imu {
+
+auto Magnetometer::Init(const std::uint8_t i2c_address) noexcept -> types::DriverStatus {
+  SENSOR_DATA_REGISTER = imu::MAGNETOMETER_MEASUREMENT_DATA;
+  REGISTER_DATA_LENGTH_IN_BYTES = 7;
```

Comment:
Magic number. Why not use REGISTER_DATA_LENGTH_IN_BYTES as a constant and set that constant to that value?

Reply:
Its used in the base class, and constant through runtime. Should I lowercase it?

Reply:
yes. please lower case it. All caps is reserved fro true const-ness. You variable is only constant through a part of the runtime (from the call of this method to it's deallocation). Const(expr) implies read only and marked for compiler optimization and unprotected thread safe and reentrancy complient

Reply:
I accidentally closed this.